invesalius/data/converters.py

```diff
@@ -192,3 +192,35 @@ def gdcm_to_numpy(image, apply_intercep_scale=True):
         return output
     else:
         return np_array
+
+def convert_custom_bin_to_vtk(filename):
+    import vtk
```

Comment:
Could you make the import at the top of the file and follow the vtk modules import, please?